spec/unit/query_components/suggest_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+
+RSpec.describe QueryComponents::Suggest do
+  context "with b variant of suggestions" do
+    it "makes the suggestion algorithm levenshtein" do
+      SPELLING_FIELD = "spelling_text".freeze
```

Comment:
Here `SPELLING_FIELD` could be `described_class::SPELLING_FIELD` so you don't need to change the test when you change the class.

Reply:
Tried this, wouldn't run, so leaving it for now.